Formula/mimic.rb

```diff
@@ -0,0 +1,31 @@
+class Mimic < Formula
+  desc "Mycroft's TTS engine, based on CMU's Flite (Festival Lite)"
```

Comment:
I don't understand this description at all (not knowing what TTS or CMU are. Please use plain English :)